react/features/noise-suppression/index.js

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Don't create an index file for the feature please.

Reply:
removed